CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 1.0.10.4 (2020-12-18)
+## Edge Agent
+### Bug Fixes
+* Rebase Edge on K8s [c94bc73](https://github.com/Azure/iotedge/commit/c94bc733c05d91acfd77c87dcaf6b701601985fb)
```

Comment:
@darobs / @veyalla - can you pls sign off on this?

Reply:
Same for all components below.

Reply:
After the fact - sorry @varunpuranik, @yophilav, but there should be no functional change in edgeHub and minor changes to non-k8s mode iotedged and edgeAgent. This was intentionally merged to bring 1.0.10 up to date with k8s public preview and vice-versa.